### PR TITLE
Load Client Tokens From Environment on Startup

### PR DESCRIPTION
**Pull request by sighphyre**
This adds the capacity to initialise a number of client tokens on startup in a similar way to how admin tokens can be created

### CODE REVIEW

File: website/docs/deploy/configuring-unleash.md [hunk 1]
```diff
@@ -89,7 +89,7 @@ unleash.start(unleashOptions);
      ```
       The tokens can be of any API token type. Note that _admin_ tokens **must** target all environments and projects (i.e. use `'*'` for `environments` and `project` and start the secret with `*:*.`).
 
-      You can also use the environment variable `INIT_ADMIN_API_TOKENS` to create API tokens on startup. This variable should be set to a comma-separated list of API tokens to initialize (for instance `*:*.some-random-string, *:*.some-other-token`). With the environment variable, all tokens will be created as admin tokens and Unleash will assign a username automatically.
+      You can also use the environment variables `INIT_ADMIN_API_TOKENS` or `INIT_CLIENT_API_TOKENS` to create API admin tokens on startup. Both variables require a comma-separated list of API tokens to initialize (for instance `*:*.some-random-string, *:*.some-other-token`). The tokens found in `INIT_ADMIN_API_TOKENS` and `INIT_CLIENT_API_TOKENS` will be created as admin and client tokens respectively and Unleash will assign a username automatically.
```
Comment (thomasheartman) [inline website/docs/deploy/configuring-unleash.md:92]: Sweet! Just one question: should this be `admin and/or client tokens`?

```suggestion
      You can also use the environment variables `INIT_ADMIN_API_TOKENS` or `INIT_CLIENT_API_TOKENS` to create API admin or client tokens on startup. Both variables require a comma-separated list of API tokens to initialize (for instance `*:*.some-random-string, *:*.some-other-token`). The tokens found in `INIT_ADMIN_API_TOKENS` and `INIT_CLIENT_API_TOKENS` will be created as admin and client tokens respectively and Unleash will assign a username automatically.
```

**Review by thomasheartman — CHANGES_REQUESTED** [review]
Thanks for updating the docs on this one 🙌🏼

File: website/docs/deploy/configuring-unleash.md (hunk 1 above)
Comment (sighphyre) [reply website/docs/deploy/configuring-unleash.md:92]: You and your eyes. That is better, thanks

**Review by thomasheartman — APPROVED** [review]
Thanks for the fix! You have my approval for the docs, but I think someone else ought to have a look at the code too. I'm not entirely sure how the system works, so don't feel quite qualified to review that just yet.

**Review by chriswk — APPROVED** [review]
LGTM

**Pull request merged**